live-rates/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import { expose, util } from '@chainlink/ea-bootstrap'
+import { execute } from './adapter'
+
+const NAME = 'Live-Rates'
+
+export = { NAME, execute, ...expose(util.wrapExecute(execute)) }
```

Comment:
We recently removed the need for wrapping External Adapters.

```suggestion
export = { NAME, execute, ...expose(execute) }
```